examples/demos/retry_apiv5_demo/main.go

```diff
@@ -41,16 +41,17 @@ func showRetryingOnMultipleCodes() error {
 	fmt.Printf("Running fake server on URL %v\n", s.URL)
 
 	demos.Step("Create Kentik API v5 client")
-	c := kentikapi.NewClient(kentikapi.Config{
+	c, err := kentikapi.NewClient(kentikapi.Config{
 		APIURL: s.URL,
 		RetryCfg: kentikapi.RetryConfig{
-			MaxAttempts:          intPtr(42),
+			MaxAttempts:          uintPtr(42),
 			MinDelay:             durationPtr(1 * time.Second),
 			MaxDelay:             durationPtr(2 * time.Second),
-			RetryableStatusCodes: []int{http.StatusTooManyRequests, http.StatusBadGateway, http.StatusServiceUnavailable},
```

Comment:
I hope we have not lost the ability to specify retryable status codes/methods. It is entirely unclear to me from the code :-/.

Reply:
We decided to simplify code for gRPC since all of these status codes in HTTP are condensed to `codes.Unavailable`. We can try to implement our own set of codes that will be translated to gRPC and HTTP if you need this feature.